Apps/APN-UIKit/APN UIKit/APN UIKit/Storage/Storage.swift

```diff
@@ -0,0 +1,135 @@
+import Foundation
+
+
+protocol StorageManager {
+    var trackUrl: String? { get set }
+    var siteId: String? { get set }
+    var apiKey: String? {get set}
+    var bgQDelay: String? {get set}
+    var bgNumOfTasks: String? {get set}
+    var isPushEnabled: Bool? {get set}
+    var isTrackScreenEnabled: Bool? {get set}
+    var isTrackDeviceAttrEnabled: Bool? {get set}
+    var isDebugModeEnabled: Bool? {get set}
+    var userEmailId: String? {get set}
+    var userName: String? {get set}
+    var deviceToken: String? {get set}
```

Comment:
This file is long and boilerplate heavy. I wonder if we can remove some of that? 

Looking at the app's code, the `StorageManager` is currently used to store the values of the `Settings` struct. 

What about the idea of refactoring the `StorageManager` to being:

```suggestion
    var settings: Settings { get set }
```

Reply:
I am OK with leaving this PR as-is for now and refactor it later. 

[I implemented the suggestion](https://github.com/customerio/customerio-ios/blob/1e5ed5e52f4589761c2b7343185922d29b8592cf/Apps/Common/Source/Store/KeyValueStore.swift) in [a draft PR](https://github.com/customerio/customerio-ios/pull/295) that we can implement into this sample app once merged.